web/src/scripts/webflow/sectorSync.mjs

```diff
@@ -84,7 +84,7 @@ const getUpdatedSectorNames = async () => {
 const getSortedSectors = async () => {
   const current = await getCurrentSectors();
   const allIndustryItem = current.find((item) => {
-    return item._id == allIndustryId;
+    return item.id == allIndustryId;
   });
```

Comment:
I updated the value as per version 2 of the Webflow API. The documentation link is in the description, along with a complete list of changes.